functions.php

```diff
@@ -25,7 +25,7 @@ function twentytwentytwo_support() {
 		add_theme_support( 'wp-block-styles' );
 
 		// Enqueue editor styles.
-		add_editor_style( get_template_directory_uri() . '/style.css' );
+		add_editor_style( 'style.css' );
```

Comment:
This undoes the commit here: https://github.com/WordPress/twentytwentytwo/pull/220#discussion_r745795293

In my testing, this prevented the stylesheet from loading in the editor.

Reply:
I can't recreate this but I'm fine with the change, based on the [docs](https://developer.wordpress.org/reference/functions/add_editor_style/#more-information) for this function.